topics/variant-analysis/tutorials/MTB_transmission/workflows/main_workflow.ga

```diff
@@ -0,0 +1 @@
: "1959cc43b414", "name": "snp_dists", "owner": "iuc", "tool_shed": "toolshed.g2.bx.psu.edu"}, "tool_state": "{\"a\": \"false\", \"b\": \"false\", \"input\": {\"__class__\": \"ConnectedValue\"}, \"k\": \"false\", \"__page__\": null, \"__rerun_remap_job_id__\": null}", "tool_version": "0.6.3+galaxy0", "type": "tool", "uuid": "843d5f8e-add2-4a0f-bc74-b5ce4e89e409", "workflow_outputs": [{"label": null, "output_name": "output", "uuid": "128e4132-ca95-432c-bd56-4d4382b6916d"}]}, "10": {"annotation": "", "content_id": "toolshed.g2.bx.psu.edu/repos/iuc/reshape2_melt/melt/1.4.2", "errors": null, "id": 10, "input_connections": {"input": {"id": 9, "output_name": "output"}}, "inputs": [{"description": "runtime parameter for tool melt", "name": "input"}], "label": "Convert distance matrix into tabular", "name": "melt", "outputs": [{"name": "output", "type": "tabular"}], "position": {"bottom": 620.2666473388672, "height": 204.9833221435547, "left": 2563.10009765625, "right": 2803.10009765625, "top": 415.2833251953125, "width": 240, "x": 2563.10009765625, "y": 415.2833251953125}, "post_job_actions": {}, "tool_id": "toolshed.g2.bx.psu.edu/repos/iuc/reshape2_melt/melt/1.4.2", "tool_shed_repository": {"changeset_revision": "317ed0ee8bf2", "name": "reshape2_melt", "owner": "iuc", "tool_shed": "toolshed.g2.bx.psu.edu"}, "tool_state": "{\"input\": {\"__class__\": \"RuntimeValue\"}, \"__page__\": null, \"__rerun_remap_job_id__\": null}", "tool_version": "1.4.2", "type": "tool", "uuid": "08c66a0f-89dd-4f66-8a72-9775cd2debae", "workflow_outputs": [{"label": null, "output_name": "output", "uuid": "53e1af40-5fb5-4470-a1bd-02f46c324dcd"}]}, "11": {"annotation": "", "content_id": "toolshed.g2.bx.psu.edu/repos/bgruening/text_processing/tp_awk_tool/1.1.2", "errors": null, "id": 11, "input_connections": {"infile": {"id": 10, "output_name": "output"}}, "inputs": [{"description": "runtime parameter for tool Text reformatting", "name": "infile"}], "label": "Filter pairwise distances with maximum of 10 SNPs", "name": "Text reformatting", "outputs": [{"name": "outfile", "type": "input"}], "position": {"bottom": 577.2999877929688, "height": 156.01666259765625, "left": 2843.10009765625, "right": 3083.10009765625, "top": 421.2833251953125, "width": 240, "x": 2843.10009765625, "y": 421.2833251953125}, "post_job_actions": {}, "tool_id": "toolshed.g2.bx.psu.edu/repos/bgruening/text_processing/tp_awk_tool/1.1.2", "tool_shed_repository": {"changeset_revision": "ddf54b12c295", "name": "text_processing", "owner": "bgruening", "tool_shed": "toolshed.g2.bx.psu.edu"}, "tool_state": "{\"code\": \"$1 != $2 && $3 <= 10\", \"infile\": {\"__class__\": \"RuntimeValue\"}, \"__page__\": null, \"__rerun_remap_job_id__\": null}", "tool_version": "1.1.2", "type": "tool", "uuid": "cce708f9-8f9a-4a03-b3fe-37d69d6a3b19", "workflow_outputs": [{"label": null, "output_name": "outfile", "uuid": "cd46c9bf-376f-4bbb-a9b4-9080e1fcadd6"}]}}, "tags": [], "uuid": "0c6b0a12-40b9-4141-b0ae-3a54c65d485a", "version": 10}
```

Comment:
Please pretty-print the JSON for easier inspection and commenting.

1. As noted on email, use the latest version of tb_variant_filter (0.3.5)
2. As noted in the comments, there is an option to bcftools consensus that can help you avoid some steps by renaming the FASTA ID from the VCF sample ID name.